scripts/populate_users_fixtures.coffee

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env coffee
+CONFIG = require 'config'
+__ = CONFIG.universalPath
+_ = __.require 'builders', 'utils'
+{ createUserWithItems } = require '../api_tests/fixtures/populate'
+
+createUserWithItems()
+.then (userCreated)->
+  _.success '#### New User available ####'
+  console.log "Your can now login with :\n\
+    - Username : #{userCreated.username} \n\
+    - Password : #{userCreated.username}"
+  process.exit 0
+.catch (err)-> console.log err
```

Comment:
this could take advantage of the "standard" way to log errors: `.catch _.Error('some label')`